server.cpp

```diff
@@ -85,23 +107,61 @@ int main(int argc, char* argv[])
         tcp::resolver resolver(io_service);
         tcp::acceptor acceptor(io_service, *resolver.resolve(tcp::resolver::query(host, port)));
 
+        namespace pls = std::placeholders;
+
+        char buff[1024];
+
+        std::ofstream fout;
```

Comment:
This variable is only used in a tiny portion of `if` inside the cycle, how about moving it there, close to the place of use?